### ISSUE TITLE
Ignore Transifex on Forked Repos

### ISSUE THREAD

**Issue by lasley** [OCA/maintainer-quality-tools#339]
Sorry in advance if I'm just being stupid here. I am getting annoying Transifex failures on my OCA forks, but from what I can see these passwords are hard coded. I even tried setting up a Transifex account but that basically just amounted to me getting a sales call.

Example:
* Failed Fork - https://travis-ci.org/laslabs/report-print-send/builds/142655502
* Same Commit Passed in OCA - https://travis-ci.org/OCA/report-print-send/builds/142655509

Is there a way within the existing framework to skip Transifex on forks? If so, what am I doing wrong? If not, is this something that a PR would be accepted for?

**Comment by moylop260** [comment]
Maybe we can validate the [REPO_SLUG-var-env](https://docs.travis-ci.com/user/environment-variables/) that start with `OCA` to avoid this issue.

For now you could disable TRANSIFEX build:

```patch
===.travis.yml
-   - TRANSIFEX="1"
+   # - TRANSIFEX="1"
```

**Comment by lasley** [comment]
I think you have a solid idea there @moylop260 !

Out of curiosity, what is your strategy here? Are you just using a Travis file that is out of sync with OCA on your forks?

**Comment by moylop260** [comment]
> @lasley Are you just using a Travis file that is out of sync with OCA on your forks?

After fork I create a commit to change badges url from `README` and change the `.travis.yml`
To create a new change I run: `git checkout -b 8.0-oca-add-x --track oca/8.0` and this branch is compatible for oca and for my fork repository

Otherwise you can use `git cherry-pick SHA` to sync just the change that you want.

**Comment by dreispt** [comment]
Sema issua as https://github.com/OCA/maintainer-quality-tools/issues/334

**Comment by dreispt** [comment]
Another example: https://travis-ci.org/dreispt/knowledge/builds/142741516

**Comment by lasley** [comment]
Thanks for the elaboration @moylop260 - I'm now torn between my laziness and my OCD here 😛 . I think I'm with @dreispt in manual action not being a solid solution though.

Oops my bad on the dup, I searched for issues but I was looking for the word Transifex. Guess I should learn to read more! Looks like this issue got a bit more traction so maybe leave it open, or do we want to switch this conversation to the predecessor?

**Comment by moylop260** [comment]
FYI I'm not agree with unnecessary builds because travis request us directly:
https://docs.travis-ci.com/user/customizing-the-build/#Build-Matrix


<img width="985" alt="screen shot 2016-07-06 at 12 06 42 pm" src="https://cloud.githubusercontent.com/assets/6644187/16627278/21a69d52-4372-11e6-9c08-2ba37c11d5b2.png">


A build with TRANSIFEX=1 where after create the container and configure to execute nothing is a bad idea in terms of `travis`

But if is a recurrent collaborators problem I don't have problem in create a `if` to avoid a red build.

**Comment by dreispt** [comment]
I don't understand your explanation: the travis.yml file id OCA projects needs fixing?
We are not supposed to change it in our PRs, so any changes ought to be in the OCA repo.

**Comment by lasley** [comment]
IMO an if statement based on the repo slug in the OCA Travis would be the best approach here. 

It would be nice if we could somehow do the logic in MQT instead in order to avoid having to update individual repos. MQT is one of those repos that I have yet to delve into though, so I'm totally clueless on implementation in that regard though 😑

**Comment by moylop260** [comment]
Please, review https://github.com/OCA/maintainer-quality-tools/pull/340

**Comment by lasley** [comment]
Closing to track in PR
